ts-declarations/md5-file/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'md5-file/promise' {
```

Comment:
I installed `@types/md5-file` in my PR. Doesn't it define this?

Reply:
it was necessary before, but it seems it works now. We have the same declaration in turtle-v2 repo